acstools/acscte.py

```diff
@@ -39,6 +40,54 @@
 __vdate__ = "13-Aug-2013"
 __all__ = ['acscte']
 
+def acscte_forward_model(input, exec_path='', time_stamps=False, verbose=False, quiet=False,
+           single_core=False, exe_args=None):
+    """
+    Run the acscteforwardmodel.e executable as from the shell.
+
+    Expect input to be ``*_blv_tmp.fits``.
+    Output is automatically named ``*_blc_tmp.fits``.
```

Comment:
I am not sure about this one. I thought it is theoretically the other way around? Maybe @jryon can double check. I wasn't involved in the API discussion.

Reply:
The output is the input with the appended suffix "_ctefmod.fits". I think the input filename doesn't technically matter, but will likely be files like FLCs and BLC_TMPs.

Reply:
Sorry, I didn't pay attention to this and just copied what was there before. TBH I wouldn't merge this if it were up to me. The bulk principle is what I was aiming for, i.e. no code dupe. The small bits here and there probably need more attention.  I left this PR open to co-edits so anyone can go for it if they like. Or copy/pull it etc. I don't really have time but, like I said, this is the sort of thing this change should look like (without rewriting it all and OOing with classes).